.editorconfig

```diff
@@ -0,0 +1,26 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+# Set default charset and indentation style
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
+
+# Max line length for python files
+[*.py]
+max_line_length = 80
```

Comment:
PEP 8 specifies 79 chars, ruff lint default is mostly consistent (it allows up to 88.

Let's start with this, but IMHO PEP 8 needs to be updated. 79 char max with modern displays is overly restrictive and discourages self documenting code techniques. I think it's reasonable to go up to 100 or even 120.